### PR TITLE
Footer and Header #13, #23

### LINKED ISSUE #13: Navbar

### LINKED ISSUE #23: Footer

### LINKED ISSUE #33: UI/UX - Dark mode support
As a user, I want dark mode support, so I can browse the website at night without hurting my eyes.

Dark theme support. Automatically set by system preferences. Otherwise, set by user preferences: toggle between light mode and dark mode.

### PR DESCRIPTION
**Pull request by MinhHoaNguyen**
closes #13, closes #23, closes #33

Dependency: Page structure #58 
https://github.com/KingSand08/LikeHome/pull/58

### CODE REVIEW

**Comment by ray-sjsu** [comment]
Fixed build errors. Moved components `header` and `footer` into separate folder.

**Comment by ray-sjsu** [comment]
Needs footer. Then, it should be good to close both issues.

File: components/layout/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                    />
+                        <span className="ml-2 text-lg font-bold ">LikeHome</span> {/* Text displayed next to image */}
+                </div>
+                {/* sign in & register */}
+                <div className="flex items-center gap-2 xl:order-2">
+                {/* Navigation Links */}
+                <nav className="flex items-center gap-8"> {/* Flex container for navigation items */}
+                        <div className="text-gray-700 hover:text-purple-600 cursor-pointer">Home</div>
+                        <div className="text-gray-700 hover:text-purple-600 cursor-pointer">Rooms</div>
+                        <div className="text-gray-700 hover:text-purple-600 cursor-pointer mr-8">Contact</div>
```
Comment (Luceium) [inline components/layout/Header.tsx:23]:
```suggestion
                        <div className="text-gray-700 hover:text-purple-600 cursor-pointer">Contact</div>
``` 
If you want a greater distance between the links and the sign in increase the gap on the parent flex container on line 19.

**Review by Luceium — CHANGES_REQUESTED** [review]
Hey, good work. This is the first significantly large PR!
I reviewed it thoroughly and added a few comments.

Some more general feedback for the entire PR are as follows:
- Some of the indentation is off. Use a formater like Prettier to fix this.
- While testing this PR I noticed that it doesn't support darkmode. This isn't a p0 and shouldn't block this PR but let's make sure that we have an issue that will track this.

**Comment by Luceium** [comment]
@MinhHoaNguyen & @ray-sjsu please make sure to respond to or resolve all comments before requesting another review.

File: app/layout.tsx [hunk 2]
```diff
@@ -1,8 +1,19 @@
 import type { Metadata } from "next";
-import { Inter } from "next/font/google";
+import { Marcellus, Urbanist } from "next/font/google";
 import "./globals.css";
+import Header from "@/components/layout/Header";
 
-const inter = Inter({ subsets: ["latin"] });
+const marcellus = Marcellus({
+  subsets: ["latin"],
+  weight: ['400'],
```
[the remark this answers is not in the document]
Comment (MinhHoaNguyen) [reply app/layout.tsx]: I changed. Don't need that code anymore.

File: components/layout/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const Footer = () => {
+    return (
+    <footer className="bg-blue-700">
+        <div className="container mx-auto">footer</div>
```
[the remark this answers is not in the document]
Comment (MinhHoaNguyen) [reply components/layout/Footer.tsx]: I tried with w-full but it doesn't look nice.

File: components/layout/Header.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+                    <Image
+                        src="/stellarHorizons.png"
+                        alt="LikeHome"
+                        width={55}
+                        height={20}
+                    />
+                        <span className="ml-2 text-lg font-bold ">LikeHome</span> {/* Text displayed next to image */}
+                </div>
+                {/* sign in & register */}
+                <div className="flex items-center gap-2 xl:order-2">
```
[the remark this answers is not in the document]
Comment (MinhHoaNguyen) [reply components/layout/Header.tsx]: code was changed. Don't use that code anymore.

**Comment by MinhHoaNguyen** [comment]
close #13, #23, #33

**Comment by Luceium** [comment]
The description of the PR needs to say closes #13, closes #23, and closes #33

**Review by Luceium — APPROVED** [review]

**Review by KingSand08 — APPROVED** [review]
Great job Raymund!

**Pull request merged**